pages/blog/async-api-spec-3.0-release.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "AsyncAPI Spec 3.0 is coming!"
+date: 2022-02-28T06:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Announcement
+  - Release
+cover: /img/posts/async-api-spec3.0-release/cover.webp
+authors:
+  - name: Harsh Mishra
+    photo: /img/avatars/harsh.webp
+    link: https://www.linkedin.com/in/harshcasper/
+excerpt: 'Announcing the start of work on AsyncAPI Spec 3.0. Checkout all the major new features coming to AsyncAPI and how you can get involved!'
+featured: false
+---
+
+AsyncAPI is a specification for describing your event-driven architecture and our specification allows you to describe and document your message-driven APIs into a human-readable format. After the last v2.3.0 release, the AsyncAPI community is inching towards a 3.0 release that will bring together multiple features, enhancements and breaking changes.
```

Comment:
> After the last v2.3.0 release, the AsyncAPI community is inching towards a 3.0 release that will bring together multiple features, enhancements and breaking changes.

this sentence sounds to me like after 2.3 there will be no 2.4. Please write it in the way that we will continue releasing spec as described in [release cadence](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md#release-cadence) and will work on 3.0 on a side